app/components/listing/MobileActionBar.jsx

```diff
@@ -2,27 +2,45 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 import { Link } from 'react-router';
+import { getResourceActions } from 'utils/ResourceActions';
 import { images } from '../../assets';
 
+import './MobileActionBar.scss';
+
+const getMobileActions = resource => {
+  const resourceActions = getResourceActions(resource);
+  const mobileActions = [
+    { ...resourceActions.edit, icon: 'edit-blue' },
```

Comment:
Non-blocking comment: I wonder if it would be better for the ResourceActions list to include both mobile and desktop icons. Something feels a little bit off about having to override the icons with the mobile, blue icons here.